### PR TITLE
chore: README improvements

### PR DESCRIPTION
**Pull request by tooomm**
## Proposed change

Improved some wordings, typos and the layout for better readability.

Result: https://github.com/tooomm/paperless-ngx/tree/patch-4#bugs

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (README)

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [ ] I have made corresponding changes to the documentation as needed.
- [ ] I have checked my modifications for any breaking changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,17 +97,19 @@ Feature requests can be submitted via [GitHub Discussions](https://github.com/pa
 
 ## Bugs
 
-For bugs please [open an issue](https://github.com/paperless-ngx/paperless-ngx/issues) or [start a discussion](https://github.com/paperless-ngx/paperless-ngx/discussions) if you have questions.
+For bugs please [open an issue](https://github.com/paperless-ngx/paperless-ngx/issues), or [start a discussion](https://github.com/paperless-ngx/paperless-ngx/discussions) in case you have questions.
 
 # Affiliated Projects
 
-Paperless has been around a while now, and people are starting to build stuff on top of it. If you're one of those people, we can add your project to this list:
+Paperless has been around a while now, and people started to build tools that interact with it. If you're one of those people, we can add your project to this list:
```
Comment (shamoon) [inline README.md:104]:
```suggestion
Paperless has been around for a while now, and people have built tools that interact with it. If you're one of those people, please reach out and we can add your project to the list. Current projects include:
```

File: README.md [hunk 2]
```diff
@@ -97,17 +97,19 @@ Feature requests can be submitted via [GitHub Discussions](https://github.com/pa
...
+  - [Paperless Share](https://github.com/qcasey/paperless_share): Share any files from your Android application with Paperless-ngx. Very simple, but works with all mobile scanning apps that allow you to share scanned documents.
+- **Desktop**
+  - [Scan to Paperless](https://github.com/sbrunner/scan-to-paperless): Scan and prepare (crop, deskew, OCR, ...) your documents for use in Paperless-ngx.
 
 # Important Note
 
-Document scanners are typically used to scan sensitive documents. Things like your social insurance number, tax records, invoices, etc. Everything is stored in the clear without encryption. This means that Paperless should never be run on an untrusted host. Instead, I recommend that if you do want to use it, run it locally on a server in your own home.
+> Document scanners are typically used to scan sensitive documents. Things like your social insurance number, tax records, invoices, etc. Everything is stored in clear text without encryption. This means that **Paperless-ngx should never be run on an untrusted host**. Instead, we recommend that if you do want to use it, **run it locally on a server in your own home**.
```
Comment (shamoon) [inline README.md:115]:
```suggestion
> Document scanners are typically used to scan sensitive documents—things like your social insurance number, tax records, invoices, etc. Everything is stored in clear text without encryption. This means that **Paperless-ngx should never be run on an untrusted host**. No guarantees are made regarding security (but we do try!) and **the safest way to run Paperless-ngx is on a local server eg in your own home**. In other words, **use at your own risk**.
```

**Review by shamoon — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -97,17 +97,19 @@ Feature requests can be submitted via [GitHub Discussions](https://github.com/pa
 
 ## Bugs
 
-For bugs please [open an issue](https://github.com/paperless-ngx/paperless-ngx/issues) or [start a discussion](https://github.com/paperless-ngx/paperless-ngx/discussions) if you have questions.
+For bugs please [open an issue](https://github.com/paperless-ngx/paperless-ngx/issues), or [start a discussion](https://github.com/paperless-ngx/paperless-ngx/discussions) in case you have questions.
```
Comment (shamoon) [inline README.md:100]:
```suggestion
For bugs please [open an issue](https://github.com/paperless-ngx/paperless-ngx/issues) or [start a discussion](https://github.com/paperless-ngx/paperless-ngx/discussions) if you have questions.
```

File: README.md [hunk 4]
```diff
@@ -101,13 +101,15 @@ For bugs please [open an issue](https://github.com/paperless-ngx/paperless-ngx/i
...
+- **Desktop**
+  - [Scan to Paperless](https://github.com/sbrunner/scan-to-paperless): Scan and prepare (crop, deskew, OCR, ...) your documents for use in Paperless-ngx.
 
 # Important Note
 
-Document scanners are typically used to scan sensitive documents. Things like your social insurance number, tax records, invoices, etc. Everything is stored in the clear without encryption. This means that Paperless should never be run on an untrusted host. Instead, I recommend that if you do want to use it, run it locally on a server in your own home.
+> Document scanners are typically used to scan sensitive documents—things like your social insurance number, tax records, invoices, etc. Everything is stored in clear text without encryption. This means that **Paperless-ngx should never be run on an untrusted host**. No guarantees are made regarding security (but we do try!) and **the safest way to run Paperless-ngx is on a local server eg in your own home**. In other words, **use at your own risk**.
```
Comment (shamoon) [inline README.md:115]:
```suggestion
> Document scanners are typically used to scan sensitive documents, things like your social insurance number, tax records, invoices, etc. Everything is stored in clear text without encryption. This means that **Paperless-ngx should never be run on an untrusted host**. No guarantees are made regarding security (but we do try!) and **the safest way to run Paperless-ngx is on a local server eg in your own home**. In other words, **use at your own risk**.
```

**Comment by shamoon** [comment]
Needs a squash =)

**Review by tooomm — COMMENTED** [review]
I see you found some more. :)
After our little ping pong match, there should be a good solution.

I gave it another run as well and tried to simplify a bit.
Not so sure about the disclaimer with security and risk you included. Is it really necessary and you want to include it for sure?

Squashed our commits to one each.

**Review by shamoon — APPROVED** [review]

**Review by stumpylog — APPROVED** [review]

**Pull request merged**